src/main/kotlin/org/rust/lang/core/resolve/NameResolution.kt

```diff
@@ -384,9 +386,15 @@ val RsFile.exportedCrateMacros: List<RsMacroDefinition>
         CachedValueProvider.Result.create(macros, PsiModificationTracker.MODIFICATION_COUNT)
     }
 
+fun Testmark.hitOnFalse(b: Boolean): Boolean {
+    if (!b)
```

Comment:
Let's join lines here. The style we use is either add braces, or put action at the same line as a condition.

Reply:
Done, I also rebased and switched to the new ways of checking Testmark in a tests.